.github/workflows/release.yml

```
@@ -13,256 +13,270 @@
+          current_version = ${CURRENT_VERSION}
           commit = False
           tag = False
-
           [bumpversion:file:main.py]
           search = __version__ = "{current_version}"
           replace = __version__ = "{new_version}"
           EOF
-
-          # Bump version
-          bumpversion $BUMP_TYPE
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)